generators/entity-client/templates/react/src/main/webapp/app/entities/entity-delete-dialog.tsx.ejs

```diff
@@ -42,6 +42,7 @@ export class <%= entityReactName %>DeleteDialog extends React.Component<I<%= ent
   handleClose = event => {
     event.stopPropagation();
     this.props.history.goBack();
+    window.location.reload();
```

Comment:
dont do this(since its not nice in a single page app, we do not want to propogate bad practices) instead just call the getAll action here and see if that helps, so that the store gets updated and list automatically gets updated.

Reply:
but seems like the action is called after the delete method and that should have updated the store.